index.js

```diff
@@ -172,7 +175,7 @@ class Peer extends stream.Duplex {
   // HACK: it's possible channel.readyState is "closing" before peer.destroy() fires
   // https://bugs.chromium.org/p/chromium/issues/detail?id=882743
   get connected () {
-    return (this._connected && this._channel.readyState === 'open')
+    return (this._connected && this._channel && this._channel.readyState === 'open')
```

Comment:
Wouldn't this always return false if you didn't set `enableDataChannel`? Is that intended?

Reply:
Good point, this was not intentional. I've updated the implementation now such that disabling data channel works as expected here.